conf/machine/p3768-0000-p3767-0003.conf

```diff
@@ -0,0 +1,24 @@
+#@TYPE: Machine
+#@NAME: Nvidia Jetson Orin Nano 8GB (P3767-0003)
+#@DESCRIPTION: Nvidia Jetson Orin Nano 8G module in P3768 carrier
+
+TEGRA_FAB ?= "300"
+TEGRA_BOARDSKU ?= "0003"
+TEGRA_BOARDREV ?= "P.1"
+
+TEGRA_BUPGEN_SPECS ?= "fab=300;boardsku=0003;boardrev=;chipsku=00:00:00:D5;bup_type=bl \
+                       fab=300;boardsku=0003;boardrev=;bup_type=kernel"
+
+TNSPEC_BOOTDEV ?= "nvme0n1p1"
+PARTITION_LAYOUT_TEMPLATE_DEFAULT ?= "flash_t234_qspi.xml"
+PARTITION_LAYOUT_TEMPLATE_DEFAULT_SUPPORTS_REDUNDANT ?= "1"
+PARTITION_LAYOUT_EXTERNAL_DEFAULT ?= "flash_l4t_t234_nvme.xml"
+TEGRAFLASH_NO_INTERNAL_STORAGE = "1"
+
+MACHINEOVERRIDES =. "orin-nano-som:"
+
+KERNEL_DEVICETREE ?= "tegra234-p3768-0000+p3767-0003-nv.dtb"
+
+TEGRA_FLASHVAR_WB0SDRAM_BCT ?= "tegra234-p3767-0000-wb0sdram-l4t.dts"
```

Comment:
This doesn't match up with what I see in the L4T kit (in `p3767.conf.common`), which uses the `-0001` variant of that file for all of the Nano SKUs (1, 3, and 5).